tests/drivers/build_all/app.overlay

```diff
@@ -102,6 +102,8 @@
 				   <&test_gpio 0 0>,
 				   <&test_gpio 0 0>,
 				   <&test_gpio 0 0>,
+				   <&test_gpio 0 0>,
+				   <&test_gpio 0 0>,
```

Comment:
@henrikbrixandersen seems like a requirement for SPI devices: https://github.com/zephyrproject-rtos/zephyr/blob/master/tests/drivers/build_all/app.overlay#L50. I added two entries because `@37` device did not add one. If not added driver fails to build (CS macros).

Reply:
@gmarull Thank you for explaining that. I was wondering why you needed to add two.